src/components/DataSubmissions/ValidationControls.tsx

```diff
@@ -172,7 +172,15 @@ const ValidationControls: FC<Props> = ({ dataSubmission, onValidate }: Props) =>
         {isValidating ? "Validating..." : "Validate"}
       </StyledValidateButton>
     ),
-    [validationType, canValidateFiles, canValidateMetadata, isValidating, isLoading]
+    [
+      validationType,
+      uploadType,
```

Comment:
This should work, but I'm wondering if we should just simplify it by adding `handleValidateFiles` as a dependency instead of `validationType` and `uploadType`? 

If `handleValidateFiles` has additional dependencies in the future, we wouldn't need to add them in here.

Reply:
Good point! I like your suggestion better